serverless/aws/lambda-core/src/main/scala/sttp/tapir/serverless/aws/lambda/AwsServerInterpreter.scala

```diff
@@ -9,7 +9,7 @@ import sttp.tapir.server.interceptor.RequestResult
 import sttp.tapir.server.interceptor.reject.RejectInterceptor
 import sttp.tapir.server.interpreter.{BodyListener, FilterServerEndpoints, ServerInterpreter}
 
-private[lambda] abstract class AwsServerInterpreter[F[_]: MonadError] {
+private[aws] abstract class AwsServerInterpreter[F[_]: MonadError] {
```

Comment:
That is required, because `sttp.tapir.serverless.aws.ziolambda.AwsZioServerInterpreter` depends on `sttp.tapir.serverless.aws.lambda.AwsServerInterpreter` and for these two the closest common package is `sttp.tapir.serverless.aws`. That is why it should be private to `[aws]` to make the code compile.